### DISCUSSION TITLE
Since no ADETAILER, how we get better faces on the txt2img? (not inpainting)

### DISCUSSION THREAD

**Discussion by ZeroCool22** [lllyasviel/Fooocus#3215] [Q&A] [answered]
In Auto's1111 we can enable ADETAILER and get better faces on txt2img process, without the need of the Inpainting extra step.

We have something in Fooocus to fix images on txt2img?

**Comment by mashb1t** [comment, marked as answer]
=> https://github.com/mashb1t/Fooocus/discussions/42 / https://github.com/lllyasviel/Fooocus/discussions/3122

Not yet scheduled for merge, but will be in an upcoming release. Feel free to use my fork instead until then, instructions how to switch without redownloading can be found [here](https://github.com/mashb1t/Fooocus?tab=readme-ov-file#switching-from-fooocus-to-this-fork).

**Reply by ZeroCool22** [reply]
> => [mashb1t#42](https://github.com/mashb1t/Fooocus/discussions/42) / #3122
> 
> Not yet scheduled for merge, but will be in an upcoming release. Feel free to use my fork instead until then, instructions how to switch without redownloading can be found [here](https://github.com/mashb1t/Fooocus?tab=readme-ov-file#switching-from-fooocus-to-this-fork).

Thx!
